### ISSUE TITLE
Doesn't work with latest crystal.

### ISSUE THREAD

**Issue by jmoriau** [stefanwille/crystal-redis#13]
Hello,

After upgrading I found out that you're still using the old syntax for function parameters definition:

```ruby
    def sort(key, by = nil, limit = nil, get = nil : Array(RedisValue)?, order = "ASC", alpha = false, store = nil)
                                                                   ^
```

The new syntax requires to invert the type and the default value:

```ruby
    def sort(key, by = nil, limit = nil, get : Array(RedisValue)? = nil, order = "ASC", alpha = false, store = nil)
                                                           ^
```

I read that it's simply a matter of running the formatter `crystal tool format` and it will automatically invert everything.

I don't think it's worth a pull request since it's just a matter of running the tool :smile:

**Comment by stefanwille** [comment]
Fixed by PR #12

**Comment by stefanwille** [comment]
Thanks for reporting the problem!
